packages/design-system/src/components/ButtonIcon/ButtonIcon.stories.mdx

```diff
@@ -0,0 +1,177 @@
+import { ArgsTable, Meta, Canvas, Story } from '@storybook/addon-docs';
+import { FigmaImage, FigmaLink, Links } from '~docs';
+import ButtonIconPrimitive from './Primitive/ButtonIconPrimitive';
+import ButtonIconVariant from './ButtonIconVariant';
+import * as Stories from './ButtonIcon.stories';
+
+<Meta title="Components/ButtonIcon" component={ButtonIconPrimitive} />
+
+<Links>
+	<FigmaLink href="https://www.figma.com/file/KuJ1PlP77uyXlfOhdniqsZ/ButtonIcon?node-id=1%3A75" />
+</Links>
+
+# ButtonIcon
+
+This component should be used when icons are meant to be clicked on.
+
+It handles the two largest usecases we have for clickable icons: actions and toggles.
+
+- Always try use when an icon is meant to be clicked on.
+- Always provide developers with copy for the tooltip.
+- If the button holds the ON / OFF state of something, then use the Toggle variants.
+- If the button is floating on top of something else, use the Floating variant.
+- Only use size XS when absolutely unavoidable.
+- Spacing between buttons should be at least 4px (XXS), 8px is best (XS).
+
+## Zoning
+
+<FigmaImage
+	src="https://www.figma.com/file/qc8oUSyVJM67gSsbTShftt/%F0%9F%93%90-Zoning?node-id=1049%3A415"
+	alt="zoning image for ButtonIcons"
+/>
+
+## Style
+
+### Variations
+
+<FigmaImage
+	src="https://www.figma.com/file/KuJ1PlP77uyXlfOhdniqsZ/ButtonIcon?node-id=3232%3A34023"
+	alt="variations image for ButtonIcon"
+/>
+
+#### Default
+
+This is the go-to ButtonIcon for most generic actions.
+
+<Canvas>
+	<Story story={Stories.Default} name="ButtonIcon" />
+</Canvas>
+
+<ArgsTable story="ButtonIcon" />
+
+#### Toggle
+
+If the button needs to hold an active / inactive state, then it is a Toggle.
+
+Don’t use Toggles in series, opt for badge dropdowns instead.
+
+<Canvas>
+	<Story story={Stories.Toggle} name="ButtonToggle" />
+</Canvas>
+
+<ArgsTable story="ButtonToggle" />
+
+#### Floating
+
+This variant is only used when the button floats on top of content.
+
+It could be it's sitting on top of a line connecting two nodes, or sticky on top of a draggable scene for instance.
+
+<Canvas>
+	<Story story={Stories.Floating} name="ButtonIconFloating" />
+</Canvas>
+
+<ArgsTable story="ButtonIconFloating" />
+
+## States
+
+ButtonIcons can display a "loading" state for asynchronous actions.
+
+### Loading
+
+Only use when necessary to avoid multiple clicks when the action is not instant.
+
+<Canvas>
+	<Story story={Stories.Loading} />
+</Canvas>
+
+### Skeleton
+
+Skeleton needs are handled by the Skeleton component.
+
+// TODO
```

Comment:
Could already be illustrated, isn't it?

Reply:
As discussed: we'll review all Skeletons when converting them to CSS modules